core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -329,6 +310,21 @@ private static ServiceRequestContext serviceRequestContext() {
         }
     }
 
+    /**
+     * Completes the {@link #whenInitialized()} with the specified value.
+     */
+    public void whenInitialized(boolean success) {
```

Comment:
How about renaming to `finishInitialization()`?

Reply:
That sounds better. 👍